content/recipes/interoperability/c4c-clinical-trials.md

```diff
@@ -0,0 +1,302 @@
+
+
+
+## Refining the initial metadata list
+The list of metadata items was reviewed by c4c partners, and those not considered cross cutting or common enough for paediatric clinical trials were removed from the list. The original list consisted of 36 items and this was reduced to 27. The following were identified for inclusion:
+
+| Considered cross cutting or common enough to be included in the metadata schema | Not considered cross cutting or common enough to be included in the Metadata Schema|
+| :------------- | :------------- |
+| Study IDs | Rare Disease (tick box yes/no) |
+| Title | Intervention Model |
+| Acronym | NCT Number |
+| Condition or Disease | Consent |
+| Therapeutic Area | Criterion |
+| Indication | Site Name |
+| Study Type | Formulation |
+| Phase | Route of Administration |
+| Funder Type |  |
+| Study Start |  |
+| Sample Size |  |
+| Study Description |  |
+| Status: Recruitment |  |
+| Study Documents |  |
+| Study Results |  |
+| Country |  |
+| Age |  |
+| Age Group |  |
+| Sex |  |
+| Ethnicity |  |
+| Race |  |
+| Additional Inclusion Criteria |  |
+| Additional Exclusion Criteria |  |
+| Outcome Measures |  |
+| Intervention/Treatment |  |
+| IMP with Orphan Designation in the Indication |  |
+| Biospecimens Retained |  |
+| Product Class |  |
+
+## Testing the metadata profile with a representative clinical trial protocol
+
+Each of the above variables were populated (where possible) with information from a clinical trial protocol. They were also mapped to the following ontologies/vocabularies:
+
+- [OMOP vocabularies](https://athena.ohdsi.org/search-terms/start)
+- [Clinical Trials Ontology](https://www.ebi.ac.uk/ols/ontologies/cto)
+- [NCI Thesaurus](https://ncit.nci.nih.gov/ncitbrowser/) 
+
+
+Example:
+
+| Term | Protocol | OMOP | Clinical Trials Ontology | NCI Thesaurus |
+| :------------- | :------------- | :------------- |:------------- | :------------- | 
+| Age Group | Child up to 15 years inclusive | 4305451 Infant 37016983 Toddler 4285883 Child 4305318 Adolescent |  NCIT:C49643 Infant & toddler NCIT:C16423 Child NCIT:C89342 Toddler NCIT:C49683 Children 2-11 years NCIT:C85405 School age child NCIT:C27954 Adolescent | C27956 Infant C89342 Toddler C16423 Child C27954 Adolescent |
+
+
+## The metadata profile in action
+
+The metadata profile created using the steps described above was used to create a metadata schema in MS Excel.
```

Comment:
Right now we used REDCap to test. Ultimately, it would be great to have a full-fledged model-driven approach in the semantic web sphere, using LinkML for the model, from which ShEx, ShaCL and entry forms are generated (e.g., via JSON-forms based on json-ld). Right now, this step was used to validate the selection of attributes, their cardinalities and allowed values, not so much their representation.

Reply:
I think we can include some screenshots from the spreadsheet that Avril created.

Reply:
I sent a cleaned up version of the spreadsheet to Danielle so that it could be included but let me know if anything else is needed from me.